pom.xml

```diff
@@ -128,146 +118,45 @@
-      </properties>
-      <distributionManagement>
-        <repository>
-          <id>ch-artifactory</id>
-          <url>${MAVEN_REPOSITORY_URL}/${artifactoryDevPublishRepo}</url>
-        </repository>
-      </distributionManagement>
-    </profile>
-    <!-- dev is the profile that is activated by default. This is only for
-      developers as it does not publish to any repository. It just downloads dependencies,
-      compiles code and builds jar for running the application and then runs the
-      tests. -->
-    <profile>
-      <id>dev</id>
-      <activation>
-        <activeByDefault>true</activeByDefault>
-      </activation>
-      <build>
-        <finalName>${project.artifactId}-${project.version}</finalName>
-        <plugins>
-          <plugin>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-maven-plugin</artifactId>
-            <configuration>
-              <mainClass>uk.gov.companieshouse.api.accounts.CompanyAccountsApplication
-              </mainClass>
-              <layout>ZIP</layout>
-            </configuration>
-            <executions>
-              <execution>
-                <goals>
-                  <goal>repackage</goal>
-                </goals>
-              </execution>
-            </executions>
-          </plugin>
-          <plugin>
-            <groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-compiler-plugin</artifactId>
-            <version>${maven-compiler-plugin.version}</version>
-            <configuration>
-              <source>${jdk.version}</source>
-              <target>${jdk.version}</target>
-              <fork>true</fork>
-              <meminitial>128m</meminitial>
-              <encoding>${project.build.sourceEncoding}</encoding>
-              <maxmem>512m</maxmem>
-            </configuration>
-          </plugin>
-          <plugin>
-            <groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-jar-plugin</artifactId>
-            <version>${maven-jar-plugin.version}</version>
-            <configuration>
-              <archive>
-                <manifest>
-                  <addClasspath>true</addClasspath>
-                </manifest>
-              </archive>
-            </configuration>
-          </plugin>
-          <plugin>
-            <groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-surefire-plugin</artifactId>
-            <version>${maven-surefire-plugin.version}</version>
-            <dependencies>
-              <dependency>
-                <groupId>org.junit.platform</groupId>
-                <artifactId>junit-platform-surefire-provider</artifactId>
-                <version>${junit-platform-surefire-provider.version}</version>
-              </dependency>
-            </dependencies>
-          </plugin>
-          <plugin>
-            <groupId>org.jacoco</groupId>
```

Comment:
There were issues removing this before I believe. Sonar test coverage dropped to zero. So it may be worth checking analysis of this PR prior to merging just in case 👍